app/views/admin/configs/show.html.erb

```diff
@@ -604,19 +604,6 @@
                 </div>
                 <input type="text" value="<%= ApplicationConfig["DEFAULT_EMAIL"] %>" class="form-control" readonly>
               </div>
-              <%= f.fields_for :email_addresses do |email_field| %>
```

Comment:
If we're only going to use the default email, and that's not changeable in the Admin, do we need to use `Settings::General` at all?

Reply:
Hmm, I think it would still make sense for it to be changeable by the admin. Right now we use `ApplicationConfig["DEFAULT_EMAIL"]` as the default but it's still changeable from the admin, I don't see why we couldn't do that for a single address too? I.e. instead of completely removing this whole section make it a single text input.

Reply:
@rhymes your suggestion makes sense, I planned to remove the multiple emails and keep the `Settings::General.email_addresses` as a first step, and remove `Settings::General.email_addresses` as a next step. 
Maybe, it would be easier to make these changes at once :thinking:  , at least for the changelog.

Reply:
@citizen428 is it changeable? It is readonly in the interface, and set to `ApplicationConfig["DEFAULT_EMAIL"]` in the `app/controllers/concerns/settings_params.rb`

Reply:
Yep it's readonly! @lightalloy I think two steps are fine!

Reply:
Ok, seems a bit unusual to me that something so important is not changeable through our main admin interface but if that's the way it is...